.github/workflows/ci-production.yml

```diff
@@ -30,34 +30,20 @@ jobs:
           yarn install
           lerna bootstrap
 
-      - name: Create .env file
-        env:
-          GOOGLE_API_KEY: ${{ secrets.GOOGLE_API_KEY }}
-          FILE_WEB3STORAGE_KEY: ${{ secrets.FILE_WEB3STORAGE_KEY }}
-          NETWORK_ID: ${{ secrets.NETWORK_ID }}
-          NETWORK_PROVIDER: ${{ secrets.NETWORK_PROVIDER }}
-          CONTRACT_ADDRESS: ${{ secrets.CONTRACT_ADDRESS }}
-          SEARCH_FORM_DISABLED: ${{ secrets.SEARCH_FORM_DISABLED }}
-        run: |
-          rm -rf ./packages/dapp/.env
-          touch ./packages/dapp/.env
-          cat > ./packages/dapp/.env << END
-          FAST_REFRESH=true
-          BROWSER=false
-          HTTPS=true
-          REACT_APP_LOG_LEVEL=debug
-          REACT_APP_MODE=production
-          REACT_APP_FILE_WEB3STORAGE_KEY=$FILE_WEB3STORAGE_KEY
-          REACT_APP_GOOGLE_API_KEY=$GOOGLE_API_KEY
-          REACT_APP_NETWORK_ID=$NETWORK_ID
-          REACT_APP_NETWORK_PROVIDER=$NETWORK_PROVIDER
-          REACT_APP_CONTRACT_ADDRESS=$CONTRACT_ADDRESS
-          REACT_APP_SEARCH_FORM_DISABLED=$SEARCH_FORM_DISABLED
-          END
-
       - name: Build dapp
         env:
           NODE_OPTIONS: --max_old_space_size=8192
+          FAST_REFRESH: true
+          BROWSER: false
+          HTTPS: true
+          REACT_APP_LOG_LEVEL: debug
+          REACT_APP_MODE: production
+          REACT_APP_GOOGLE_API_KEY: ${{ secrets.GOOGLE_API_KEY }}
+          REACT_APP_FILE_WEB3STORAGE_KEY: ${{ secrets.FILE_WEB3STORAGE_KEY }}
+          REACT_APP_NETWORK_ID: ${{ secrets.NETWORK_ID }}
+          REACT_APP_NETWORK_PROVIDER: ${{ secrets.NETWORK_PROVIDER }}
+          REACT_APP_CONTRACT_ADDRESS: ${{ secrets.CONTRACT_ADDRESS }}
+          REACT_APP_SEARCH_FORM_DISABLED: ${{ secrets.SEARCH_FORM_DISABLED }}
         run: |
           rm -rf ./docs
```

Comment:
suggest insert `export` here for absolute debugging.

Reply:
Why? We definitely know that env variables are being applied ...

Just took a look of how you do it here:

https://github.com/windingtree/lpms-server/blob/main/.github/workflows/release.yml

```
    - name: Release
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        NPM_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
      run: npx semantic-release
```

You don't use `export` @mfw78 .